src/reducers/index.js

```diff
@@ -3,10 +3,14 @@ import { signUp } from "./signUp";
 import { listReducer } from "./listReducer";
 import { combineReducers } from "redux";
 import { plantReducer } from "./plantReducer";
+import { deletePlantReducer } from ".deletePlantReducer";
+import { updateReducer } from ".updateReducer";
```

Comment:
`import updateReducer from ".updateReducer"

You want to bring in a **default export** ... not a **named export** ... so no curly brackets needed.